x-pack/plugins/ml/common/constants/messages.ts

```diff
@@ -51,6 +51,12 @@ export const getMessages = once((docLinks?: DocLinksStart) => {
   return {
     categorizer_detector_missing_per_partition_field: {
       status: VALIDATION_STATUS.ERROR,
+      heading: i18n.translate(
```

Comment:
Do all the messages need a `heading`? If so, some of the are missing one:

```cardinality_model_plot_high index_fields_invalid  index_fields_valid missing_summary_count_field_name skipped_extended_tests estimated_mml_greater_than_max_mml mml_greater_than_max_mml time_field_invalid datafeed_preview_failed```

Reply:
No, only the ones with a link